app/styles/components.scss

```diff
@@ -150,3 +150,4 @@
 @import 'components/school/visualizer-session-type-vocabularies';
 @import 'components/school/visualizer-session-type-vocabulary';
 
+@import 'components/courses/list-item';
```

Comment:
it feels like the courses-list scss should also move here? `import 'components/courses/list';`?

Reply:
i didn't touch that on purpose, since it pertains to the controller template, and not the actual component. it's a mess.

Reply:
:heavy_check_mark: